Src/MoneyFox.Ui/Views/Statistics/CategorySummary/StatisticCategorySummaryPage.xaml

```diff
@@ -95,6 +96,12 @@
                         </Grid>
                     </DataTemplate>
                 </CollectionView.ItemTemplate>
+                <CollectionView.Footer>
+                    <StackLayout VerticalOptions="Center" HorizontalOptions="End" Margin="0,10, 10, 0" Padding="12">
+                        <Label Text="{Binding TotalExpense, StringFormat={extensions:Translate TotalExpenseTemplate}, Converter={StaticResource AmountFormatConverter}}" Style="{StaticResource TextSubheadSecondary}" />
+                        <Label Text="{Binding TotalIncome, StringFormat={extensions:Translate TotalRevenueTemplate}, Converter={StaticResource AmountFormatConverter}}}" Style="{StaticResource TextSubheadSecondary}" Margin="0,5,0,0" />
```

Comment:
Do those templates already exist in the `Translations.resx`?

Reply:
I see you have reverted your change in Translations.resx but I didn't check in your changes. Will add it.

Reply:
ah, yeah. that change slipped into the master a bit too early. I wanted to add unit tests first myself. sorry for the confusion.

Reply:
Added the required templates